### PR TITLE
Socialmedia icons added

### PR DESCRIPTION
**Pull request by Sruthi-Jayaraj**
please accept the social media icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,5 +13,13 @@ <h1>Merge Master and Feature</h1>
 
 	<h1>Positive Message from Developer1</h1>
 
+	<div>
+		<a href="https://www.facebook.com/bootsnipp"><i id="social-fb" class="fa fa-facebook-square fa-3x social"></i></a>
```
Comment (SruthiJayaraj91) [inline index.html]: remove the bootsnipp

File: index.html [hunk 2]
```diff
@@ -12,5 +13,13 @@ <h1>Merge Master and Feature</h1>
...
+        <a href="https://twitter.com/bootsnipp"><i id="social-tw" class="fa fa-twitter-square fa-3x social"></i></a>
+        <a href="https://plus.google.com/+Bootsnipp-page"><i id="social-gp" class="fa fa-google-plus-square fa-3x social"></i></a>
+        <a href="mailto:bootsnipp@gmail.com"><i id="social-em" class="fa fa-envelope-square fa-3x social"></i></a>
```
Comment (SruthiJayaraj91) [inline index.html]: remove bootsnipp

File: index.html (hunk 1 above)
Comment (Sruthi-Jayaraj) [reply index.html]: Okay

File: index.html (hunk 2 above)
Comment (Sruthi-Jayaraj) [reply index.html]: okay

**Comment by SruthiJayaraj91** [comment]
Thank you

**Pull request merged**